src/GraphQLController.php

```diff
@@ -88,7 +88,7 @@ protected function queryContext(string $query, ?array $params, string $schema)
 
     public function graphiql(Request $request, string $schema = null): View
     {
-        $graphqlPath = '/'.config('graphql.prefix');
+        $graphqlPath = config('graphql.domain').'/'.config('graphql.prefix');
```

Comment:
it would not work automatic if it just used the root url,

URL::to('/');

Reply:
uh oh, double slash `//` bug or so, right?